docs/contracts.rst

```diff
@@ -469,9 +469,20 @@ View Functions
 
 Functions can be declared ``view`` in which case they promise not to modify the state.
 
+The following statements are considered modifying the state:
+
+#. Writing to state variables.
+#. :ref:`Emitting events. <events>`
+#. :ref:`Creating other contracts <creating-contracts>`
+#. Using ``selfdestruct``.
+#. Sending Ether via calls.
```

Comment:
This is redundant as a function cannot be marked view/pure and payable, but I guess it cannot hurt atm (since it is not enforced).